db/migrate/20220713141153_create_post.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePost < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :Title
+      t.string :Text
+
+      t.timestamps
+    end
```

Comment:
- [ ] in the posts table there should be a comments counter and likes counter as well.